src/fsharp/AttributeChecking.fs

```diff
@@ -248,21 +248,21 @@ let MethInfoHasAttribute g m attribSpec minfo  =
 
 
 /// Check IL attributes for 'ObsoleteAttribute', returning errors and warnings as data
-let private CheckILAttributes (g: TcGlobals) isByrefLikeTyconRef cattrs m = 
+let private CheckILAttributes (g: TcGlobals) cattrs m = 
```

Comment:
Feels like we can apply the filtering here?

Reply:
We can. And honestly might be the easiest path to just get what we want to work.

Reply:
Ok, I think I remember now. We can't just have the filtering be applied here, we also need it for NameResolution.